rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -73,6 +73,7 @@ rabbitmq_ulimit: 65535
 # since memory utilization is primarily driven by the objects placed in memcached.
 memcached_connections: 16384
 
+#TODO (evrardjp): Remove this when ocata
 # Keystone overrides
 keystone_token_driver: "sql"
```

Comment:
wouldn't the persistence driver just be ignored if you are using fernet provider...which is fine?